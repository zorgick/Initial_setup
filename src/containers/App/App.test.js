import React from 'react'
import {App} from './index'
import {shallow, mount} from 'enzyme'

describe('Testing App container...', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(<App loadGreetings={() => {}} hi="Hi from test" />)
  })

  test('should render App container', () => {
    expect(wrapper).toMatchSnapshot()
  })

  test('should handle click on the button', () => {
    console.log(wrapper.props())
  })
})
