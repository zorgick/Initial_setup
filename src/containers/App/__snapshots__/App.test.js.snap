// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App container... should render App container 1`] = `
<App
  hi="Hi from test"
  loadGreetings={[Function]}
>
  <section>
    <h1>
      Hi from test
    </h1>
    <button
      onClick={[Function]}
    >
      Handshake
    </button>
  </section>
</App>
`;
